juriscraper/opinions/united_states/state/neb.py

```diff
@@ -11,33 +11,29 @@ def __init__(self, *args, **kwargs):
         self.status = "Published"
 
     def _process_html(self):
-        path = {
-            "group": '//div[contains(@class, "view-grouping")]',
-            "date": './/span[contains(@class, "date-display-single")]',
-            "row": ".//tbody/tr",
-        }
-        for group in self.html.xpath(path["group"]):
-            date_element = group.xpath(path["date"])
-            if not date_element:
+        for table in self.html.xpath(".//table"):
+            date_tags = table.xpath("preceding::time[1]/text()")
+            if not date_tags:
+                print("NO DATE TAGS")
                 continue
-            date = date_element[0].text_content()
-            for row in group.xpath(path["row"]):
-                url = row.xpath(".//td[3]//a/@href")
-
-                # skip rows without url in third cell
-                if not url:
+            for row in table.xpath(".//tr"):
+                cells = row.xpath(".//td")
+                # make sure its a valid row - and not the ending of the table
+                if len(cells) != 3 or cells[0].xpath(".//text()") is None:
                     continue
```

Comment:
I think you should drop a logger call here too, just in case it is skipping somewhere it shouldn't be

Reply:
I think a better xpath will resolve this.